pkg/chain/ethlike/block_counter.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 )
 
+// BlockCounter represents a block counter
```

Comment:
We need to add dots at the ends of all these comments we're adding in this PR.

Reply:
Done in f1c11be35fce9272ea39e30ad61d1ee28b03ec80.